deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -26,8 +24,8 @@
     client_cert "#{ENV['K8S_METADATA_FILTER_CLIENT_CERT']}"
     client_key "#{ENV['K8S_METADATA_FILTER_CLIENT_KEY']}"
     bearer_token_file "#{ENV['K8S_METADATA_FILTER_BEARER_TOKEN_FILE']}"
-    cache_size "#{ENV['K8S_METADATA_FILTER_BEARER_CACHE_SIZE']}"
-    cache_ttl "#{ENV['K8S_METADATA_FILTER_BEARER_CACHE_TTL']}"
+    cache_size "#{ENV['K8S_METADATA_FILTER_CACHE_SIZE']}"
+    cache_ttl "#{ENV['K8S_METADATA_FILTER_CACHE_TTL']}"
```

Comment:
Can we also change lines 33-34 to use the env var, as well as in the `metrics.conf`, for consistency? Once we make the major release we can use the templating this way as it is cleaner but until then I think we should stay consistent with the use of env vars

Reply:
Sure